gdcdictionary/schemas/observation.yaml

```
@@ -773,3 +773,19 @@ properties:
 
   subjects:
     $ref: "_definitions.yaml#/to_one"
+
+# Prop originated for Longcovid and Monkeypox data
+
+  observations_icd10:
+    description: >
+      zero, one or more ICD-10 codes for encounter, separated by commas
```

```suggestion
      zero, one or more ICD-10 codes for encounter
```
if they are stored in an array, then they are not a comma-separated string